src/lang.js

```diff
@@ -2,6 +2,8 @@ var Messages = require('./messages');
 
 require('./lang/en');
 
+var require_method = require;
```

Comment:
Could you make this camelCased to be consistent with the rest of the code?

Reply:
I am fine with it @skaterdav85 but do agree this should be camelCased